tests/integration/test_yarn.py

```diff
@@ -0,0 +1,55 @@
+import logging
+from pathlib import Path
+
+import pytest
+
+from . import utils
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    "test_params",
+    [
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/cachi2-yarn-berry.git",
+                ref="76131443f7472916b316954c957259ee4cee97a7",
+                packages=({"path": ".", "type": "yarn"},),
+                check_vendor_checksums=False,
+                expected_exit_code=0,
+                expected_output="All dependencies fetched successfully",
+            ),
+            marks=pytest.mark.xfail,  # temporary
+            id="offline-cache",
+        ),
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/cachi2-yarn-berry.git",
+                ref="a37898af6d5f2fa4335e179444fad1ca7ad74613",
+                packages=({"path": ".", "type": "yarn"},),
+                check_vendor_checksums=False,
+                expected_exit_code=0,
+                expected_output="All dependencies fetched successfully",
+            ),
+            marks=pytest.mark.xfail,  # temporary
+            id="zero-installs",
```

Comment:
Let's follow the same `id` naming as other integration tests: `yarn_{test_case}` or `yarnberry_{test_case}`. This makes it so that you can run all the yarn tests with `tox -e integration -- -k yarn`